src/constants/index.ts

```diff
@@ -216,4 +216,59 @@ export const PANGOLIN_TOKENS_REPO_RAW_BASE_URL = `https://raw.githubusercontent.
 
 export type LogoSize = 24 | 48;
 
+export const ANALYTICS_PAGE = 'https://info.pangolin.exchange';
+
+export enum BETA_MENU_LINK {
```

Comment:
why this export BETA_MENU_LINK here?

Reply:
this variable is used in Watchlist/Coinchart